init.lua

```diff
@@ -1,9 +1,12 @@
-package.path = package.path..";/home/NIDAS/?.lua;/home/NIDAS/lib/?.lua"
+package.path = package.path .. ";/home/NIDAS/?.lua;/home/NIDAS/lib/?.lua"
```

Comment:
Really?

Reply:
I opened the file and reformatted it, It's one of the changes, I see nothing wrong with it